packages/runtime/container-runtime/package.json

```diff
@@ -143,6 +143,9 @@
       "RemovedInterfaceDeclaration_IPendingRuntimeState": {
         "forwardCompat": false,
         "backCompat": false
+      },
+      "ClassDeclaration_ScheduleManager": {
```

Comment:
Ideally, we stop exporting this class out of package, and refactor UTs in functional-tests to leverage mock DeltaManager (and thus move it into same package).
If this is hard, let's at least mark this class as @internal. While I think it does not help yet directly, we should follow up with work for all the compat workflows to ignore internal things (this has some risks as it pushes more responsibility to engineers, but I think that's fine)

Reply:
https://github.com/microsoft/FluidFramework/pull/12050 I've opened a task to refactor that test.